packages/terra-popup/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Changed
+  * Added event handlers to call `stopPropagation()` on key and focus events on PopupContent for `terra-popup` support in `terra-compact-interactive-list` cell.
```

Comment:
can this feature be used outside of compact interactive list?

Reply:
It works everywhere, doesn't matter inside of CIL or outside of it, as outside of the compact interactive list it makes no difference. I just wanted to provide a reason why it was added, as it makes no difference for all the cases when popoup is outside of CIL (or any other component that has key navigation).

Reply:
Then I would reword this to be something generic like:

```suggestion
  * Added event handlers to call `stopPropagation()` on key and focus events on PopupContent in other components.
```